src/app/app-routing.module.ts

```diff
@@ -1,14 +1,19 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { FeedComponent } from './components/feed/feed.component';
+import { AuthorizedComponent } from './components/authorized/authorized.component';
+import { FeedComponent } from './components/authorized/feed/feed.component';
 
 const routes: Routes = [
-    { path: '', redirectTo: 'feed', pathMatch: 'full' },
-    { path: 'feed', component: FeedComponent}
+    {
+        path: 'authorized',
+        component: AuthorizedComponent,
+        children: [{ path: 'feed', component: FeedComponent }],
+    },
 ];
```

Comment:
There is no path for registration that is already in master. But i guess it will be there after resolving conflicts.

Reply:
Yes.